pkg/server/auth/server.go

```diff
@@ -109,19 +112,30 @@ func NewAuthServer(ctx context.Context, cfg AuthConfig) (*AuthServer, error) {
 		if err != nil {
 			return nil, fmt.Errorf("could not create provider: %w", err)
 		}
+		// Set isOidcEnabled now we have a valid provider
+		isOidcEnabled = true
 	}
 
 	return &AuthServer{cfg, provider}, nil
 }
 
+// SetOidcEnabled is intended for test use only
+func SetOidcEnabled(newVal bool) {
+	isOidcEnabled = newVal
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
This is not how we give feedback.

We give a suggestion, and explain to the author why we prefer our way to theirs. We do not slam a "RTFM" in their heads.

Reply:
I'm happy to change to `isOIDCEnabled` but IMO it's less readable and doesn't match the existing method names